generate man page from markdown

alex19EP opened the pull request:
note: this requires #30

williamh commented:
This is also automagic, so please avoid that. Use a user option to control whether the dependency is required.
pandoc is going to be pretty heavy for source-based distros because it will pull in all of haskell to build it.
Do you mind checking out [go-md2man](http://github.com/cpuguy83/go-md2man)?

alex19EP commented:
> This is also automagic, so please avoid that. Use a user option to control whether the dependency is required.

fixed with option man

> pandoc is going to be pretty heavy for source-based distros because it will pull in all of haskell to build it.


I switched to ronn.
